efopen/ef_template_resolver.py

```diff
@@ -413,8 +415,12 @@ def render(self):
       # resolve and replace symbols
       for symbol in template_symbols:
         resolved_symbol = None
+
+        # Don't resolve symbols that are provided as skippable
+        if symbol.split(',')[0] in self.skip_symbols:
+          resolved_symbol = "NONE"
```

Comment:
Instead of resolving it to`NONE`, we should have it resolve to something that makes it very clear that it got skipped.
```suggestion
          resolved_symbol = "SKIPPED_SYMBOL"
```